tests/components/media_player/test_samsungtv.py

```diff
@@ -301,3 +306,54 @@ def test_turn_on(self):
         self.device._mac = "fake"
         self.device.turn_on()
         self.device._wol.send_magic_packet.assert_called_once_with("fake")
+
+
+@pytest.fixture
+def samsung_mock():
+    """Mock samsungctl."""
+    with patch.dict('sys.modules', {
+        'samsungctl': MagicMock(),
+    }):
+        yield
+
+
+async def test_play_media(hass, samsung_mock):
+    """Test for play_media."""
+    asyncio_sleep = asyncio.sleep
+    sleeps = []
+
+    async def sleep(duration, loop):
+        sleeps.append(duration)
+        await asyncio_sleep(0, loop=loop)
+
+    with patch('asyncio.sleep', new=sleep):
+        device = SamsungTVDevice(**WORKING_CONFIG)
+        device.hass = hass
+
+        hass.async_add_job = mock.Mock()
```

Comment:
I don't think we can do this, since we should await this. I think we need to create a function that appends the argument etc to a list so we can check them and returns a task that can be awaited.

Check if there are any useful helpers in tests/common.py.

Reply:
An alternative would be to patch at a later stage.